### DISCUSSION TITLE
Allow two videos to be played in parallel

### DISCUSSION THREAD

**Discussion by pesce-volante** [smplayer-dev/smplayer#747] [Ideas]
Hi, I practice car racing.   It is very valuable to analyze my races and my laps by looking the videos with smplayer, and great to measure the the time up to the milliseconds.

Now, I would simply like to play two videos in two parallel windows and instances of smplayer, in order to compare two laps, for example.   Unfortunately this is apparently intentionally impossible.

Could we have a configuration allowing the possibility to open two instances of smplayer in parallel (there are some use cases where this is useful, e.g. compare crash-tests, science, etc.).

I guess, it should be very easy to code.

Many thanks

**Comment by Raza-0007** [comment]
If I understand you correctly, you need two videos playing side-by-side in SMPlayer?

There is an mpv command that can open two videos in one SMPlayer instance simultaneously. Just copy paste the following command in mpv options field, changing the path of the test videos to the exact path of your two videos:

Options>Preferences>Advanced>MPlayer/mpv>Options:>

--lavfi-complex="[vid1][vid2]hstack[vo]" D:\test1.mp4 --external-file=D:\test2.mp4

Where
Video 1 path= D:\test1.mp4
Video 2 path=D:\test2.mp4

After entering the above command just restart SMPlayer and hit Play.

You can choose which audio plays by choosing the respective track from SMPlayer audio track selection.

Hopefully this solves your issue, if I understood the issue correctly.

**Comment by pesce-volante** [comment]
Hi Raza-0007, sorry for having wasted your time.  
My problem is actually solved by going under Preferences-->Interface-->uncheck "Use only one running instance of SMPlayer".
I am happy now, I can run two videos in parallel and compare them, up to the millisecond.
SMPlayer is great!
Thanks
